harness/determined/deploy/aws/cli.py

```diff
@@ -296,6 +297,20 @@ def deploy_aws(command: str, args: argparse.Namespace) -> None:
     if not args.no_preflight_checks:
         check_quotas(det_configs, deployment_object)
 
+    if not deployment_object.exists():
```

Comment:
If this goes into a helper library, I'm not sure where it should live (not enough opinions about our packaging layout, I know what I'd do personally though).